app/models.py

```diff
@@ -340,3 +340,50 @@ class UserSkill(db.Model): #pylint: disable=no-init,too-few-public-methods
     user_id = Column(types.Integer, ForeignKey('users.id'), nullable=False)
 
     constraint = UniqueConstraint('user_id', 'name')
+
+class Event(db.Model):
+    '''
+    An event that shows up in the activity feed for a deployment.
+    '''
+
+    __tablename__ = 'events'
+    id = Column(types.Integer, autoincrement=True, primary_key=True)  #pylint: disable=invalid-name
+    created_at = Column(types.DateTime(), default=datetime.datetime.now)
+    updated_at = Column(types.DateTime(), default=datetime.datetime.now,
+                        onupdate=datetime.datetime.now)
+    type = Column(types.String)
+
+    user_id = Column(types.Integer, ForeignKey('users.id'))
```

Comment:
Yeah, I'm not sure about it either... Another alternative is to have a subclass of `Event` called `UserEvent` which most (if not all) events subclass from. Though if we don't require that users be affiliated with events, we might have to add a `deployment` field to the `Event` model too.